.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Setup CocoaPods
         uses: maxim-lobanov/setup-cocoapods@v1
         with:
-          version: 1.14.3
+          version: 1.5.2
```

Comment:
Why did we downgrade to such a lowest version?

Reply:
What was the GitHub actions issue you were referring to?

Reply:
Yes, it was related to the GitHub Action issue. Basically, one of the GitHub actions was failing due to some unknown issue. 
I'll try to look into this again.

Reply:
I've replaced it with a different action. The previous action was not supporting the latest version if the Cocoapod.